sys-apps/fd/fd-6.3.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 2017-2018 Gentoo Foundation
```

Comment:
I don't think that's the correct header. You always put the original start date here.